doc/operation.md

```diff
@@ -15,6 +15,12 @@
 The one set of files that is required but not generated by this code are the 37h
 threshold binaries provided by Tom Mote. These are checked in to this repository.
 
+> NOTE:
+> The path commands are for users working in a bash environment! For csh users, 
+> the commands are:
+	 setenv PYTHONPATH /location where antarctica_today lives
+	python antactica_today/program.py
```

Comment:
```suggestion
> the commands are:
>    setenv PYTHONPATH /location where antarctica_today lives
>    python antactica_today/program.py
```

The code wasn't rendering as expected. I think this will fix it.